stdlib/public/core/BridgeObjectiveC.swift

```diff
@@ -369,7 +369,7 @@ public struct AutoreleasingUnsafeMutablePointer<Pointee /* TODO : class */>
     /// Retrieve the value the pointer points to.
     @_transparent get {
       // We can do a strong load normally.
-      return unsafeBitCast(self, to: UnsafeMutablePointer<Pointee>.self).pointee
+      return UnsafeMutablePointer(mutating: self).pointee
```

Comment:
As for the label, it compiles, and this is what the fixit suggested. Whether we need a mutable pointer is a very good question, that I don't know the answer for.